libs/ledgerjs/packages/types-live/src/bridge.ts

```diff
@@ -194,6 +194,25 @@ export interface AccountBridge<
    * @param {Account} account - The original account object.
    */
   assignFromAccountRaw?: (accountRaw: R, account: A) => void;
+  /**
+   * This function mutates the 'tokenAccountRaw' object in-place to add any extra fields that the coin may need to set.
+   * It is called during the serialization mechanism
+   *
+   * @param {TokenAccount} tokenAccount - The original token account object.
+   * @param {TokenAccountRaw} tokenAccountRaw - The token account in its serialized form.
+   */
+  assignToTokenAccountRaw?: (tokenAccount: TokenAccount, tokenAccountRaw: TokenAccountRaw) => void;
```

Comment:
This may not be useful, we don't have to do this with other token supporting integrations